src/views/Combine.vue

```diff
@@ -621,8 +621,13 @@ export default {
       // and clean
       layerSet = cleanLayerSet(layerSet)
       layerSet = selectVariantsInLayerSet(layerSet, this.scenarioIds)
+
+      // before showing new notifications, clear existing ones
+      this.$store.commit('clearNotifications')
```

Comment:
I noticed that the notifications are also stored in the [notifcationBar](https://github.com/Deltares/liwo-static/blob/713c0f1dcca5f064c56c1d7be2a6122f1480a47d/src/components/NotificationBar.vue#L81). Does that still work properly? It loads notifications when it is mounted.